core/esmf-test-aspect-models/src/main/resources/valid/samm_2_0_0/org.eclipse.esmf.test/1.0.0/AspectWithConstrainedSet.ttl

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix : <urn:samm:org.eclipse.esmf.test:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+
+:AspectWithConstrainedSet a samm:Aspect ;
+   samm:preferredName "Test Aspect"@en ;
+   samm:description "This is a test description"@en ;
+   samm:see <http://example.com/> ;
+   samm:properties ( :testProperty ) ;
+   samm:operations ( ) .
+
+:testProperty a samm:Property ;
+   samm:preferredName "Test Property"@en ;
+   samm:description "This is a test property."@en ;
+   samm:see <http://example.com/> ;
+   samm:see <http://example.com/me> ;
+   samm:characteristic :TestTrait .
+
+:TestTrait a samm-c:Trait;
+    samm-c:baseCharacteristic :TestSet;
+    samm-c:constraint :TestSetConstraint.
+
+:TestSet a samm-c:Set ;
+   samm:preferredName "Test Set"@en ;
+   samm:description "This is a test set."@en ;
+   samm:see <http://example.com/> ;
+   samm:dataType xsd:string .
+
+:TestSetConstraint a samm-c:LengthConstraint;
+    samm:preferredName "Company ID Set Constraint"@en;
+    samm:description "Constraint for defining a non-empty set of company identifiers as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en;
```

Comment:
This is oddly specific for a test model; please remove this description and the samm:see